src/util-unittest-helper.c

```diff
@@ -942,6 +942,27 @@ int UTHParseSignature(const char *str, bool expect)
     PASS;
 }
 
+/** \brief writes the contents of a buffer into a file */
+int UTHbufferToFile(const char * name, const uint8_t *Data, size_t Size) {
```

Comment:
style: argument names should be all lower case

Reply:
ok